app/ai/voice/agents/breeze_buddy/agent/turn_strategies/factories/analyzer_factory.py

```
@@ -0,0 +1,47 @@
+        Raises:
+            ImportError: If analyzer dependencies are not installed
+            ValueError: If analyzer type is not supported
+        """
+        params = params or {}
+
+        # Only support local_smart_turn_v3
+        if analyzer_type != TurnAnalyzerType.LOCAL_SMART_TURN_V3:
+            raise ValueError(
+                f"Only '{TurnAnalyzerType.LOCAL_SMART_TURN_V3}' analyzer is supported. "
+                f"Got: {analyzer_type}"
+            )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)